src/main/java/io/github/minecraftcursedlegacy/api/client/TextureRegistry.java

```diff
@@ -0,0 +1,19 @@
+package io.github.minecraftcursedlegacy.api.client;
+
+import io.github.minecraftcursedlegacy.impl.client.TextureRegistryImpl;
+
+public class TextureRegistry {
```

Comment:
possibly use a different name to emphasize the fact that it isn't an actual registry. Perhaps `Textures`